source/index.html.md

```diff
@@ -146,52 +142,98 @@ certain triggers may pass in.
 
-We are interested in the message parameter. We simply check if it contains the word we are interested in, and if so, we use the `ChatLib` utility to send a message only visible to the player. You can read more about the `ChatLib` utility [here](#ChatLib)
+We are interested in the message parameter. We simply check if it contains the word we are interested in, and if so, we use the `ChatLib` utility to send a message only visible to the player. You can read more about the `ChatLib` utility [here](#chatlib)
+
+## Command Trigger
+
+Another one of the most common triggers is the Command trigger. This allows the user to make custom commands of their choosing.
+
+```js
+register("command", (user) => {
+  ChatLib.chat("Hi " + user);
+}).setName("mycommand");
+```
+
+Commands are one of the few triggers that do not have event as a parameter. This example creates a command which can be called through `/mycommand`. The arguments of the function are the arguments the user types into the command. If the user types `/mycommand kerbybit`, then the function will be triggered, and `Hi kerbybit` will be printed into chat.
+
+<aside class="warning">
+  Forge has a bug which will make commands with capital letters as the base to not fire. Set the full command name lowercase to avoid the issue.
+</aside>
 
 ## Module Organization
 
 > ES6 style import syntax (preferred)
 
 ```js
-// This imports a function from WhereAmI's index file, you could specify additional files i.e. WhereAmI/otherfile
-// You can also import from other files within the current module using local file paths
-// ...such as ./folder/file (where folder resides within WhereAmI)
-import playerLocation from 'WhereAmi';
+import playerLocation from "WhereAmi";
 ```
 
+This imports a function from WhereAmI's index file, you could specify additional files i.e. WhereAmI/otherfile
+You can also import from other files within the current module using local file paths such as `./folder/file` (where folder resides within WhereAmI).
+
 > Node style require syntax
 
 ```js
-// This imports a function from another module, called WhereAmI, to get a players location
-let playerLocation = require('WhereAmI/index');
+const playerLocation = require("WhereAmI/index");
 ```
 
-When making a module, it is important to know how ct.js loads your files so you can organize them efficiently. When your module is loaded, only the file specified as the entry in `metadata.json` is loaded. Any other code you want to run must be imported, through the `require` syntax, or ES6 style `import` syntax.
+This imports a function from another module, called WhereAmI, to get a players location.
+
+When making a module, it is important to know how ChatTriggers loads your files so you can organize them efficiently. When your module is loaded, only the file specified as the entry in `metadata.json` is loaded. Any other code you want to run must be imported, through the `require` syntax, or ES6 style `import` syntax.
```

Comment:
Honestly let's just remove `require` from the docs altogether, since `import`/`export` is preferred now.